docs/fr/users/07_Frequently_Asked_Questions.md

```diff
@@ -4,7 +4,7 @@ Il est possible que nous n'ayons pas répondu à toutes vos questions dans les p
 
 Bien entendu, le ```/i``` n'est pas là pour faire joli ! Il s'agit d'une question de performances et de praticité :
 
-* Cela permet de servir les icônes, images, styles, scripts sans cookie. Sans cela, ces fichiers seraient souvent re-téléchargés, en particulier lorsque Persona ou le formulaire de connexion sont utilisés. De plus, les requêtes vers ces ressources seraient plus lourdes.
+* Cela permet de servir les icônes, images, styles, scripts sans cookie. Sans cela, ces fichiers seraient souvent re-téléchargés, en particulier lorsque le formulaire de connexion est utilisé. De plus, les requêtes vers ces ressources seraient plus lourdes.
```

Comment:
@aledeg @Alkarex Please double check this one.

Reply:
Perfect 🥇